docs/user-guide/styling-your-docs.md

```diff
@@ -100,6 +100,30 @@ the [Bootswatch] project.
 
 ![Yeti](http://bootswatch.com/yeti/thumbnail.png)
 
+## Third party themes
+
+There are many themes provides for the community like:
```

Comment:
This is a nitpick, but I wouldn't use "many" here. Also, "provided" not "provides".